source/sdk/threading/native/ConditionWin32.ooc

```diff
@@ -0,0 +1,87 @@
+version(windows) {
+
+	import ../Thread
+	import native/win32/[types, errors]
+	import MutexWin32
+	import structs/ArrayList
+
+	include windows
+
+	CreateEvent: extern func (...) -> Handle
+	SetEvent: extern func (Handle) -> Bool
+	CloseHandle: extern func (Handle) -> Bool
+	INFINITE: extern Long
+	WAIT_OBJECT_0: extern Long
```

Comment:
Okay, no problem. I just don't want to litter the code with UGLY_ALL_CAPS_CONSTANTS if it can be avoided :)